config/cluster.yaml.template

```diff
@@ -0,0 +1,11 @@
+# base cluster settings
```

Comment:
can this be flattened? the yaml parser will probably return something that begins with 'base', but i dont think we need that